.github/workflows/dependabot-auto-merge.yml

```diff
@@ -14,8 +14,7 @@ jobs:
           app-id: ${{ secrets.FETCH_METADATA_ACTION_AUTOMATION_APP_ID }}
           private-key: ${{ secrets.FETCH_METADATA_ACTION_AUTOMATION_PRIVATE_KEY }}
 
-      - name: Check out code
-        uses: actions/checkout@v4
+      - uses: actions/checkout@v4
```

Comment:
NAB: We should likely prefer to use a pinned SHA version for safety here since this is automerging PRs ( and rely on Dependabot to keep this up to date 😅 )

Reply:
`actions/checkout` has been migrated to immutable actions, so it's now safe to ping directly to a human semver value. 🎉